src/custom/constants/subsidy.ts

```diff
@@ -0,0 +1,10 @@
+export const COW_SUBSIDY = [
+  ['(v)COW balance', 'Fee discount'],
+  ['0', 0],
+  ['100', 10],
+  ['1000', 15],
+  ['10000', 25],
+  ['100000', 35],
+]
+
+export const SUBSIDY_INFO_MESSAGE = 'As a (v)COW token holder you will be eligbible for a fee discount.'
```

Comment:
Same as this, this is content. Shouldn't be in a constant imo. I know, u want to refactor some repetition. I don't think a bit of repetition in content is that bad, and if its in a constant should be in a constant in the component, not a global one